recipe/meta.yaml

```diff
@@ -6,33 +6,31 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/openfast/openfast/archive/v{{ version }}.tar.gz
-  sha256: 94da085f1d8861a93ab32b6b39335a403d182be0b5f7ff48f63d5805eb7f2b2c
```

Comment:
why would you remove the usage of the url and sha256? this is the best practice at conda forge.